drupal/tpl/drupal-rc.yaml

```diff
@@ -0,0 +1,66 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-rc.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: drupal
+  labels:
+    app: drupal
+    provider: drupal-server
+    heritage: bitnami
+spec:
+  replicas: 1
+  selector:
+    app: drupal
+    provider: drupal-server
+    version: 8.1.1-r0
+  template:
+    metadata:
+      labels:
+        app: drupal
+        provider: drupal-server
+        version: 8.1.1-r0
+        heritage: bitnami
+    spec:
+      containers:
+      - name: drupal
+        image: bitnami/drupal:8.1.1-r0
+        env:
+        - name: MARIADB_HOST
+          value: "mariadb"
+        - name: MARIADB_PORT
+          value: "3306"
+        - name: MARIADB_PASSWORD
+          value: "{{ .mariadbPassword | quote }}"
```

Comment:
Remove the quotes, similar to https://github.com/bitnami/charts/blob/master/mariadb/tpl/mariadb-rc.yaml